python/GafferUI/RampPlugValueWidget.py

```diff
@@ -34,16 +35,121 @@
 #
 ##########################################################################
 
-import Gaffer
-import GafferUI
 import imath
 import IECore
 
-from Qt import QtCore
-from Qt import QtGui
+import Gaffer
+import GafferUI
 
 class RampPlugValueWidget( GafferUI.PlugValueWidget ) :
 
+	def __init__( self, plug, **kw ) :
+
+		self.__splineWidget = GafferUI.RampWidget()
```

Comment:
__splineWidget -> __rampWidget?

It would have been nice if this widget refactor was in a separate commit I think - there's more going on than the simple renaming in the rest of the files. The rename of the class will also need calling out in Changes.md.